### DISCUSSION TITLE
links

### DISCUSSION THREAD

**Discussion by giscus** [siuze/NotionNext#3] [Announcements]
# links

单页不会显示在博客列表，会在导航添加一个菜单。

https://notion.siuze.top/links

**Comment by Guanchishan** [comment]
來這所躒蜀躒

**Comment by Guanchishan** [comment]
記得你以前是說在努力把寫的東西都搬到Notion，看來今年應該是完工啦？👀
我還有些東西殘留在靜態博客上，也得找時間搬運搬運了hhhh
